test/lit.cfg

```diff
@@ -151,3 +151,8 @@ else:
 # POSIX runtime feature
 if config.enable_posix_runtime:
   config.available_features.add('posix-runtime')
+
```

Comment:
@andreamattavelli You pushed just before I was about to push a commit implementing this.
I've rebased and done pushed a commit that cleans this up a bit.